package.json

```diff
@@ -75,7 +75,7 @@
     "sw-precache": "5.0.0",
     "swagger-js-codegen": "1.7.12",
     "ts-node": "2.1.0",
-    "tslint": "4.4.2",
+    "tslint": "^4.5.1",
```

Comment:
If possible, we should always be using the same patch versions as well. I try to stick to `4.5.1`, for example.